Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.6.6
```

Comment:
Is this Ubuntu?

Reply:
Debian I believe

Reply:
Could you try switching this to `2.6.6-alpine3.12` pls :)